### DISCUSSION TITLE
Installing multiple plugins

### DISCUSSION THREAD

**Discussion by LBF38** [Flow-Launcher/Flow.Launcher#1449] [Ideas]
Another idea would be to be able to select multiple plugins at once and then, installing them all together.
This would avoid going back and forth between the Flow settings and the restarting process of the app launcher.

What do you think of this feature ?

**Comment by onesounds** [comment]
There have been several similar requests, but it is a little difficult to implement. be under consideration
I think the backup/restore function is more useful because the plug-in setting must be config by user as well when installing the plug-in at first.  If there is a backup restore function, I don't think it's necessary to install multiple plug-ins. 
we are only reviewing it because it is equally difficult to implement. 😂

**Reply by LBF38** [reply]
More than the backup/restore function, the idea would be to select every plugins we want to install and then, install them all it once to restart the app only once. Therefore, this is useful for the first time we use it and install plugins. 
This feature would be in pair with viewing what plugins are installed or not.
In a broader view, it can be part of a plugin management system in the app. But I think it isn't part of the core features of the Flow Launcher.

**Reply by onesounds** [reply]
I agree that there is a need. I am not particularly opposed to it.
It is just difficult to easily confirm whether it is implemented or not.
I will put it in the work list, but as you know, it is difficult to reflect quickly because we don't have enough workers. 😂 I hope you understand.

**Reply by LBF38** [reply]
Thank you for taking it into account in the app development ! ^^ 
I know you're working hard to build the app and I'll try to help you on my free time if I can.
